### DISCUSSION TITLE
segment error

### DISCUSSION THREAD

**Discussion by guyingi** [apache/doris#20712] [A - General / Q&A] [answered]
centos7
openjdk-11
doris-1.2.4.1
I got segment error when write date into be, after this error be backend broken and shutdown。 
whether it's a bug

` 
I0612 16:35:54.537747 23110 tablet_sink.cpp:1237] finished to close olap table sink. load_id=9a5c4b56097f4ef7-8a1f09649380efb5, txn_id=16667, node add batch time(ms)/wait execution time(ms)/close time(ms)/num: {10003:(32)(0)(35)(1)} 
*** Query id: 9a5c4b56097f4ef7-8a1f09649380efb5 ***
*** Aborted at 1686558954 (unix time) try "date -d @1686558954" if you are using GNU date ***
*** Current BE git commitID: Unknown ***
*** SIGSEGV address not mapped to object (@0x0) received by PID 22701 (TID 0x7f1557744700) from PID 0; stack trace: ***

0# doris::signal::(anonymous namespace)::FailureSignalHandler(int, siginfo_t*, void*) at /root/doris/be/src/common/signal_handler.h:420
 1# os::Linux::chained_handler(int, siginfo_t*, void*) in /usr/lib/jvm/java-11-openjdk-11.0.19.0.7-1.1.al7.x86_64/lib/server/libjvm.so
 2# JVM_handle_linux_signal in /usr/lib/jvm/java-11-openjdk-11.0.19.0.7-1.1.al7.x86_64/lib/server/libjvm.so
 3# signalHandler(int, siginfo_t*, void*) in /usr/lib/jvm/java-11-openjdk-11.0.19.0.7-1.1.al7.x86_64/lib/server/libjvm.so
 4# 0x00007F1683AD8400 in /lib64/libc.so.6
 5# je_arena_dalloc_promoted at ../src/arena.c:1604
 6# __GI__dl_deallocate_tls in /lib64/ld-linux-x86-64.so.2
 7# __free_stacks in /lib64/libpthread.so.0
 8# __deallocate_stack in /lib64/libpthread.so.0
 9# pthread_join in /lib64/libpthread.so.0
10# doris::ThreadJoiner::join() at /root/doris/be/src/util/thread.cpp:544
11# doris::Thread::join() at /root/doris/be/src/util/thread.cpp:300
12# doris::stream_load::OlapTableSink::close(doris::RuntimeState*, doris::Status) at /root/doris/be/src/exec/tablet_sink.cpp:1257
13# doris::stream_load::VOlapTableSink::close(doris::RuntimeState*, doris::Status) at /root/doris/be/src/vec/sink/vtablet_sink.cpp:653
14# doris::PlanFragmentExecutor::open_vectorized_internal() at /root/doris/be/src/runtime/plan_fragment_executor.cpp:338
15# doris::PlanFragmentExecutor::open() at /root/doris/be/src/runtime/plan_fragment_executor.cpp:261
16# doris::FragmentExecState::execute() at /root/doris/be/src/runtime/fragment_mgr.cpp:261
17# doris::FragmentMgr::_exec_actual(std::shared_ptr<doris::FragmentExecState>, std::function<void (doris::PlanFragmentExecutor*)>) at /root/doris/be/src/runtime/fragment_mgr.cpp:508
18# std::_Function_handler<void (), doris::FragmentMgr::exec_plan_fragment(doris::TExecPlanFragmentParams const&, std::function<void (doris::PlanFragmentExecutor*)>)::{lambda()#1}>::_M_invoke(std::_Any_data const&) at /var/local/ldb-toolchain/include/c++/11/bits/std_function.h:291
19# doris::ThreadPool::dispatch_thread() at /root/doris/be/src/util/threadpool.cpp:543
20# doris::Thread::supervise_thread(void*) at /root/doris/be/src/util/thread.cpp:455
21# start_thread in /lib64/libpthread.so.0
22# clone in /lib64/libc.so.6

./start_be.sh: 行 246: 22701 段错误(segment error)               ${LIMIT:+${LIMIT}} "${DORIS_HOME}/lib/doris_be" "$@" 2>&1 < /dev/null`

**Comment by yagagagaga** [comment, marked as answer]
Don't use jdk11, use jdk8.

**Reply by guyingi** [reply]
thanks, perfect solution
